Functions/Administration/xASL_adm_GetDeprecatedFields.m

```diff
@@ -111,6 +111,7 @@
     'M0PositionInASL4D'             'modules'   'asl'                                'M0PositionInASL4D';...
     'HadamardType'                  'modules'   'asl'                                'HadamardType';...
 	'ApplyQuantification'           'modules'   'asl'                                'ApplyQuantification';...
+    'MergeSessionList'              'modules'   'asl'                                'MergeSessionList';...
```

Comment:
You just changed this name in the last commit, I don't know why, but if you change the name, I would consider the order of words. Starting with List makes it easier to sort variables by their type (==list), starting by Session makes it easier to sort variables by what they effect (==subject,session,etc) but starting with Merge to me has little advantages; but I'm sure you had an idea to change the variable name? :)

Reply:
I can change this to any other name.